### ISSUE TITLE
Damage Graph is gone?

### ISSUE THREAD

**Issue by rdc4687** [d07RiV/d3planner#491]
The graph in the bottom right hand corner is not showing, has it been removed?  If so, please bring it back!

**Comment by MerLock-ui** [comment]
Agreed. The graph on the bottom right was very useful in determining which gear piece upgrade may contribute to the biggest change. Being able to track changes with the graph as I played with my gear helped a lot. Please bring it back if possible. Thanks!
